srcCxx/makefile_rti

```diff
@@ -35,14 +35,16 @@ SOURCES  += $(RTI_SRC_DIR)/ShapeTypeSupport.cxx $(RTI_SRC_DIR)/ShapeTypePlugin.c
 LIBS     +=  -L$(NDDSHOME)/lib/$(TARGET_ARCH) \
              -lnddssecurityzd -lnddscppzd -lnddsczd -lnddscorezd \
              -L$(RTI_OPENSSLHOME)/$(TARGET_ARCH)/debug/lib \
-             -lssl -lcrypto $(SYSLIBS)
+             $(RTI_OPENSSLHOME)/$(TARGET_ARCH)/debug/lib/libssl.a \
+             $(RTI_OPENSSLHOME)/$(TARGET_ARCH)/debug/lib/libcrypto.a \
+             $(SYSLIBS)
 DEFINES  += -DRTI_CONNEXT_DDS -DRTI_UNIX -DRTI_LINUX 
 INCLUDES += -I$(NDDSHOME)/include -I$(NDDSHOME)/include/ndds 
 #End of RTI
        
 
 OBJS    = $(SOURCES:%.cxx=%.o)
-EXEC    = rti_connext_dds_6.1.0_linux
+EXEC    = rti_connext_dds_7.3.0_linux
```

Comment:
Update https://github.com/omg-dds/dds-security/blob/master/README.md as well. Right now it mentions `rti_connext_dds_5.2_linux`.